tests/unit/all.js

```diff
@@ -827,4 +827,31 @@ describe('code snippet example', () => {
       assert.equal(callCount, 1);
     });
   })
+
+
+  describe('isFunction', () => {
```

Comment:
It will be better if there are some test cases to prove that `isFunction()` is returning `true` or `false` in certain cases instead of having similar behavior with lodash/underscore only?

Reply:
Makes sense to me, I'll suggest an edit.